# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieLogging.cpp
# Commit: d01a1a055fec
# Full Hash: d01a1a055fece12b906b938954f984bd69278ef7
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2025-04-30 20:49:53
# Regressor Bug: 1960824
# File Overlap Count: 1
# Description:
#   Bug 1960824 - Introduce nsICookie::SAMESITE_UNSET, r=edgul,webdriver-reviewers,extension-reviewers,cookie-reviewers,valentin,Sasha,robwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D246638
# ==============================================================================

diff -r 307ff391dd02 -r d01a1a055fec netwerk/cookie/CookieLogging.cpp
--- a/netwerk/cookie/CookieLogging.cpp	Wed Apr 30 08:53:12 2025 +0000
+++ b/netwerk/cookie/CookieLogging.cpp	Wed Apr 30 08:59:18 2025 +0000
@@ -129,9 +129,7 @@
     MOZ_LOG(gCookieLog, LogLevel::Debug,
             ("is httpOnly: %s\n", aCookie->IsHttpOnly() ? "true" : "false"));
     MOZ_LOG(gCookieLog, LogLevel::Debug,
-            ("sameSite: %s - rawSameSite: %s\n",
-             SameSiteToString(aCookie->SameSite()),
-             SameSiteToString(aCookie->RawSameSite())));
+            ("sameSite: %s\n", SameSiteToString(aCookie->SameSite())));
     MOZ_LOG(
         gCookieLog, LogLevel::Debug,
         ("schemeMap %d (http: %s | https: %s | file: %s)\n",